main.cpp

```diff
@@ -243,238 +242,229 @@ struct Resturant
-    //4) inputs (int)
     int inputs = 5;
-    //5) outputs (int)
     int outputs = 4;
-    //1) play cds
     void playCds();
-    //2) play records
     void playRecords();
-    //3) play radio
     void playRadio();
 };
-/*
-Thing 8) remote
-5 properties:
-    1) brand (std::string)
-    2) model number (int)
-    3) range (float)
-    4) function selector (int)
-    5) batteries needed (int)
-3 things it can do:
-    1) select inputs
-    2) adjust volume
-    3) select mode
- */
+
+void Receiver::playCds()
+{
+    
+}
+
+void Receiver::playRecords()
+{
+    
+}
+
+void Receiver::playRadio()
+{
+    
+}
+
 struct Remote
 {
-    //1) brand (std::string)
     std::string brand = "Pioneer";
-    //2) model number (int)
     int modelNum = 1600;
-    //3) range (float)
     float range = 135.4f;
-    //4) function selector (int)
     int selector = 5;
-    //5) batteries needed (int)
     int batteries = 2;
-    //1) select inputs
     void inputs();
-    //2) adjust volume
     int adjustVol(int volUp, int volDwn, int currVol);
-    //3) select mode
     void mode();
 };
-/*
-Thing 9) turntable
-5 properties:
-    1) brand (std::string)
-    2) model number (int)
-    3) belt speed (float)
-    4) outputs (int)
-    5) needle weight (float)
-3 things it can do:
-    1) plays records
-    2) plays at 33 or 45 rpms
-    3) adjusts for anti-skip
- */
+
+void Remote::inputs()
+{
+    
+}
+
+int Remote::adjustVol(int volUp, int volDwn, int currVol)
+{
+    return currVol += volUp - volDwn;
+}
+
+void Remote::mode()
+{
+    
+}
+
 struct Turntable
 {
-    //1) brand (std::string)
     std::string brand = "Mercury";
-    //2) model number (int)
     int modelNum = 1400;
-    //3) belt speed (float)
     float beltSpeed = 25.4f;
-    //4) outputs (int)
     int outputs = 4;
-    //5) needle weight (float)
     float needleWeight = 0.0043f;
-    //1) plays records
     void playRecord();
-    //2) plays at 33 or 45 rpms
     bool rpms(bool thirtThree);
-    //3) adjusts for anti-skip
     void skipAdjust();
 };
-/*
-Thing 10) stereo system
-5 properties:
-    1) speakers
-    2) amp
-    3) receiver
-    4) remote
-    5) tuntable
-3 things it can do:
-    1) play records
-    2) play radio
-    3) record audio
- */
+
+void Turntable::playRecord()
+{
+    
+}
+
+bool Turntable::rpms(bool thirtThree)
+{
+    return thirtThree;
+}
+
+void Turntable::skipAdjust()
+{
+    
+}
+
 struct StereoSystem
 {
-    //1) speakers
     Speakers speaker;
-    //2) amp
     Amp amp;
-    //3) receiver
     Receiver receiver;
-    //4) remote
     Remote remote;
-    //5) tuntable
     Turntable turntable;
-    //1) play records
     Receiver playRecord();
-    //2) play radio
     Receiver playRadio();
-    //3) record audio
     Receiver recordAudio();
 };
+Receiver StereoSystem::playRecord()
+{
+    return receiver;
+}
+
+Receiver StereoSystem::playRadio()
+{
+    return receiver;
+}
+
+Receiver StereoSystem::recordAudio()
+{
+    return receiver;
+}
```

Comment:
Why do these functions return instances of a `Receiver` class? 
**It does not make sense** why they would return such an object. 
The work that your functions do should make logical sense, even if they aren't doing very much of anything.
- [ ] revise, make it make sense.